code/modules/mechs/mech_interaction.dm

```diff
@@ -333,22 +333,41 @@
 				MC.return_diagnostics(user)
 		return
 
+	else if(istype(I, /obj/item/stack/nanopaste))
+		var/obj/item/stack/nanopaste/paste = I
+		if(paste.amount < 2)
+			to_chat(user, SPAN_WARNING("You need at least 5 nanopastes pieces in order to repair damage."))
```

Comment:
```suggestion
		if(paste.amount < 2)
			to_chat(user, SPAN_WARNING("You need at least 2 charges in order to repair damage."))
```
"Charges" may not be the best word, but it flows a bit better. Feel free to swap it with something similar.